src/app/containers/CpsRelatedContent/index.test.jsx

```diff
@@ -88,13 +88,10 @@ describe('CpsRelatedContent', () => {
       pageData,
     );
 
-    renderRelatedContent({
+    const { getByText } = renderRelatedContent({
       content: transformedPromos,
     });
 
-    const renderedTime = document
-      .querySelector(`aside time`)
-      .getAttribute('datetime');
-    expect(renderedTime).toBe('2009-02-14');
+    expect(getByText('February 2009', { exact: false })).not.toBeNull();
```

Comment:
expect(getByText('February 2009')).toBeInTheDocument();

Would this work? I'm expecting @jroebu14 to suggest it :)

Reply:
I had that originally, but `toBeInTheDocument` is seemingly not part of react testing library - it comes from another package: https://stackoverflow.com/questions/56547215/react-testing-library-why-is-tobeinthedocument-not-a-function

Reply:
Interesting. Thanks for explaining :)